Sources/NIOExtras/PCAPRingBuffer.swift

```diff
@@ -24,6 +25,7 @@ public class NIOPCAPRingBuffer {
     private var pcapCurrentBytes: Int
     private let maximumFragments: Int
     private let maximumBytes: Int
+    private let lock = Lock()
```

Comment:
Yeah I think this change isn't necessary: this class was fine as it was, it doesn't need to be sendable or thread-safe.